server/src/test/java/org/kaazing/gateway/server/config/parse/ServiceDefaultsTest.java

```diff
@@ -26,7 +30,7 @@ public void testDefaultConnectOptions() throws Exception {
                 .connectOption("udp.interface", "en0")
                 .connectOption("tcp.transport", "socks://localhost:8000")
                 .connectOption("http.keepalive", "disabled")
-                .connectOption("http.keepaliveTimeout", "5sec")
+                .connectOption("http.keepalive.timeout", "5sec")
```

Comment:
See also kaazing/roadmap#180
(Story for standardizing names)

Is there Doc Impact here?